SPECS/javapackages-bootstrap/javapackages-bootstrap.spec

```diff
@@ -0,0 +1,372 @@
+# Exclude automatically generated requires on java interpreter which is not
+# owned by any package
+%global         __requires_exclude ^/usr/lib/jvm/java
+
+# Don't run OSGi dependency generators on private (bundled) JARs
+%global         __requires_exclude_from \\.jar$
+%global         __provides_exclude_from \\.jar$
+
+%global mavenHomePath %{_datadir}/%{name}
+%global metadataPath %{mavenHomePath}/maven-metadata
+%global artifactsPath %{_javadir}
+%global launchersPath %{_libexecdir}/%{name}
+
+Name:           javapackages-bootstrap
+Version:        1.5.0
```

Comment:
why not using latest (1.9.0) ? 1.5.0 is from 07/2021

Reply:
this is the version where fedora was using openjdk-11. 
the newer version uses openjdk-17. Hence had to use the older version.